### PR TITLE
Include Decision Notes in proposal

### LINKED ISSUE #2: Parse "Decision Notes" URL from proposals
We currently don't parse the "Decision Notes" field

### PR DESCRIPTION
**Pull request by digoreis**
-Include Decision Notes in Proposal;
closes #2

### CODE REVIEW

**Review by unnamedd — APPROVED** [review]

File: Sources/DecisionNote.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+//
+//  License
+//  Copyright © 2016 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+import Foundation
+
+/// Represents a review manager for a proposal.
+public struct DecisionNote {
```
Comment (jessesquires) [inline Sources/DecisionNote.swift]: let's rename `DecisionNotes`

File: Sources/DecisionNote.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    public let title: String
```
Comment (jessesquires) [inline Sources/DecisionNote.swift]: how about `text` instead?

File: Sources/DecisionNote.swift [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+//
+//
+//  License
+//  Copyright © 2016 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+import Foundation
+
+/// Represents a review manager for a proposal.
```
Comment (jessesquires) [inline Sources/DecisionNote.swift]: fix doc comment

File: Sources/DecisionNote.swift [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+extension DecisionNote: CustomStringConvertible {
+    public var description: String {
+        return title
+    }
+}
+
+extension DecisionNote: Equatable {
+    public static func ==(lhs: DecisionNote, rhs: DecisionNote) -> Bool {
+        return lhs.title == rhs.title
```
Comment (jessesquires) [inline Sources/DecisionNote.swift]: this should also compare URL

File: Sources/DecisionNote.swift [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+extension DecisionNote: Comparable {
+    public static func < (lhs: DecisionNote, rhs: DecisionNote) -> Bool {
+        return lhs.title < rhs.title
+    }
+}
+
+extension DecisionNote: Hashable {
+    public var hashValue: Int {
+        return title.hashValue
```
Comment (jessesquires) [inline Sources/DecisionNote.swift]: also include `URL`

**Review by jessesquires — CHANGES_REQUESTED** [review]
Thanks @digoreis ! 🎉 

Just a few things to address!

File: Sources/DecisionNote.swift (hunk 3 above)
Comment (digoreis) [reply Sources/DecisionNote.swift]: 😞

File: Sources/DecisionNote.swift (hunk 2 above)
Comment (digoreis) [reply Sources/DecisionNote.swift]: Ok!

File: Sources/DecisionNote.swift (hunk 4 above)
Comment (digoreis) [reply Sources/DecisionNote.swift]: In this case I think we'd better use a URL, it makes more sense. Correct?

**Comment by jessesquires** [comment]
Thanks for updating @digoreis ! 👍 Looks good now, except there are conflicts to resolve since merging #16 😅

**Comment by jessesquires** [comment]
thanks so much @digoreis ! 💯

**Pull request merged**